integration-tests/tests/register-user.js

```diff
@@ -14,7 +14,7 @@ const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
   await verifyAccessibility(page)
   await click(page, clickable({ textContains: 'Register' }))
   await click(page, clickable({ textContains: 'Accept' }), { timeout: 90000 })
-  await findText(page, 'To get started, Create a New Workspace')
+  await findText(page, 'Welcome to Terra Community Workbench')
```

Comment:
Somehow, this test was ending on the workspaces page. But accepting the terms of service should navigate to the home page.

https://github.com/DataBiosphere/terra-ui/blob/d1825e54ba29c45cedf0e9c09d4637181a55c16d/src/pages/TermsOfService.js#L58